### PR TITLE
delete post

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Delete post
## Reference Requirements
- Task ID: #135

## Reference Isuees
#{issue_id}

## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: public/js/posts.js [hunk 1]
```diff
@@ -33,7 +35,7 @@ function contentReview(data) {
         for (var i = 1; i <= k; i++) {
             stars += '<i class="fa fa-star"></i>'; 
         };
-        reviews += '<div class="review">\
+        reviews += '<div class="review" id="review_post'+value.id+'">\
```
Comment (at-haytran) [inline public/js/posts.js]: thêm khoảng trắngsau dấu +

File: public/js/posts.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
 var bookId = window.location.pathname;
+
```
Comment (at-haytran) [inline public/js/posts.js]: xóa khoảng trống

**Comment by at-haytran** [comment]
LGTM

**Comment by at-vietpham** [comment]
LGTM 🔨

**Pull request merged**